README.md

```diff
@@ -416,20 +416,21 @@
 
 ### Modules
 
-* <a name="consistent-module-structure"></a>
-  Use a consistent structure in module definitions.
-  <sup>[[link](#consistent-module-structure)]</sup>
+* <a name="use-import-alias-require"></a>
+  Use a consistent structure when calling `use`/`import`/`alias`/`require`: call
+  them in this order and group multiple calls to each of them.
+  <sup>[[link](#use-import-alias-require)]</sup>
 
   ```elixir
-  @compile :inline_list_funcs
-
   use GenServer
 
-  require Logger
+  import Bitwise
+  import Kernel, except: [length: 1]
 
-  alias Kernel.Typespec
+  alias Mix.Utils
+  alias MapSet, as: Set
```

Comment:
Since `Set` is the module name of another module in Elixir (not matter it has been softdeprecated), this could be confusing and considered as not good practice. I would use a different examples such as `alias Macro.Env, as: Env`. wdy-guys-t?